### ISSUE TITLE
TA_FUNC_UNST_HT_TRENDMODE is defined 2 times

### ISSUE THREAD

**Issue by femtotrader** [mrjbq7/ta-lib#103]
Hello,

I'm working on a Julia wrapper for TALib https://github.com/femtotrader/TALib.jl

I noticed something odd... not sure if it's a bug (but it might be)

See https://github.com/mrjbq7/ta-lib/blob/43fa277989c43986e926fe1a24685c05c248382f/talib/libta_lib.pxd

```python
    ctypedef int TA_FuncUnstId
    TA_FuncUnstId TA_FUNC_UNST_ADX = 0
    TA_FuncUnstId TA_FUNC_UNST_ADXR = 1
    TA_FuncUnstId TA_FUNC_UNST_ATR = 2
    TA_FuncUnstId TA_FUNC_UNST_CMO = 3
    TA_FuncUnstId TA_FUNC_UNST_DX = 4
    TA_FuncUnstId TA_FUNC_UNST_EMA = 5
    TA_FuncUnstId TA_FUNC_UNST_HT_DCPERIOD = 6
    TA_FuncUnstId TA_FUNC_UNST_HT_DCPHASE = 7
    TA_FuncUnstId TA_FUNC_UNST_HD_PHASOR = 8
    TA_FuncUnstId TA_FUNC_UNST_HT_SINE = 9
    TA_FuncUnstId TA_FUNC_UNST_HT_TRENDLINE = 10
    TA_FuncUnstId TA_FUNC_UNST_HT_TRENDMODE = 11
    TA_FuncUnstId TA_FUNC_UNST_HT_TRENDMODE = 12
    TA_FuncUnstId TA_FUNC_UNST_KAMA = 13
    TA_FuncUnstId TA_FUNC_UNST_MAMA = 14
    TA_FuncUnstId TA_FUNC_UNST_MFI = 15
    TA_FuncUnstId TA_FUNC_UNST_MINUS_DI = 16
    TA_FuncUnstId TA_FUNC_UNST_MINUS_DM = 17
    TA_FuncUnstId TA_FUNC_UNST_NATR = 18
    TA_FuncUnstId TA_FUNC_UNST_PLUS_DI = 19
    TA_FuncUnstId TA_FUNC_UNST_PLUS_DM = 20
    TA_FuncUnstId TA_FUNC_UNST_RSI = 21
    TA_FuncUnstId TA_FUNC_UNST_STOCHRSI = 22
    TA_FuncUnstId TA_FUNC_UNST_T3 = 23
    TA_FuncUnstId TA_FUNC_UNST_ALL = 24
    TA_FuncUnstId TA_FUNC_UNST_NONE = -1
````

to compare with

https://github.com/stoni/ta-lib/blob/6edc8d665f145ca7eb19c6992191e0c4f0b61ec0/c/include/ta_defs.h

```C
ENUM_BEGIN( FuncUnstId )
    /* 000 */  ENUM_DEFINE( TA_FUNC_UNST_ADX, Adx),
    /* 001 */  ENUM_DEFINE( TA_FUNC_UNST_ADXR, Adxr),
    /* 002 */  ENUM_DEFINE( TA_FUNC_UNST_ATR, Atr),
    /* 003 */  ENUM_DEFINE( TA_FUNC_UNST_CMO, Cmo),
    /* 004 */  ENUM_DEFINE( TA_FUNC_UNST_DX, Dx),
    /* 005 */  ENUM_DEFINE( TA_FUNC_UNST_EMA, Ema),
    /* 006 */  ENUM_DEFINE( TA_FUNC_UNST_HT_DCPERIOD, HtDcPeriod),
    /* 007 */  ENUM_DEFINE( TA_FUNC_UNST_HT_DCPHASE, HtDcPhase),
    /* 008 */  ENUM_DEFINE( TA_FUNC_UNST_HT_PHASOR, HtPhasor),
    /* 009 */  ENUM_DEFINE( TA_FUNC_UNST_HT_SINE, HtSine),
    /* 010 */  ENUM_DEFINE( TA_FUNC_UNST_HT_TRENDLINE, HtTrendline),
    /* 011 */  ENUM_DEFINE( TA_FUNC_UNST_HT_TRENDMODE, HtTrendMode),
    /* 012 */  ENUM_DEFINE( TA_FUNC_UNST_KAMA, Kama),
    /* 013 */  ENUM_DEFINE( TA_FUNC_UNST_MAMA, Mama),
    /* 014 */  ENUM_DEFINE( TA_FUNC_UNST_MFI, Mfi),
    /* 015 */  ENUM_DEFINE( TA_FUNC_UNST_MINUS_DI, MinusDI),
    /* 016 */  ENUM_DEFINE( TA_FUNC_UNST_MINUS_DM, MinusDM),
    /* 017 */  ENUM_DEFINE( TA_FUNC_UNST_NATR, Natr),
    /* 018 */  ENUM_DEFINE( TA_FUNC_UNST_PLUS_DI, PlusDI),
    /* 019 */  ENUM_DEFINE( TA_FUNC_UNST_PLUS_DM, PlusDM),
    /* 020 */  ENUM_DEFINE( TA_FUNC_UNST_RSI, Rsi),
    /* 021 */  ENUM_DEFINE( TA_FUNC_UNST_STOCHRSI, StochRsi),
    /* 022 */  ENUM_DEFINE( TA_FUNC_UNST_T3, T3),
               ENUM_DEFINE( TA_FUNC_UNST_ALL, FuncUnstAll),
               ENUM_DEFINE( TA_FUNC_UNST_NONE, FuncUnstNone) = -1
ENUM_END( FuncUnstId )
```

so we have
```
    TA_FuncUnstId TA_FUNC_UNST_HT_TRENDMODE = 11
    TA_FuncUnstId TA_FUNC_UNST_HT_TRENDMODE = 12
```

`TA_FUNC_UNST_HT_TRENDMODE` is defined with 2 differents values (11 and 12).
According TALib C code Value 12 should be used for `TA_FUNC_UNST_KAMA`
It seems to lead to an offset (after 11)

Your opinion @mrjbq7 ?

Kind regards

**Comment by mrjbq7** [comment]
Good catch.  Fixed in f4adb01f20b9ae687628dc2daad50f9f29aff9f5.

**Comment by femtotrader** [comment]
Thanks for fixing this.
